ACD-839 (#58)
Actindo-AG/Documentation: Core1Platform/03_InnovativeArchitecture.md

### Flexible APIs

Companies can accelerate innovation with modern APIs (application programming interfaces) and easy-to-use low-code development environment. With over 500 API endpoints, you can integrate existing back-end systems or additional third-party applications. You can extend the platform to your needs, regardless of the front end of your customer's devices. 

### Cloud technology

Instead of worrying about maintaining, connecting, and updating a monolithic ERP software, you can use a fail-safe and scalable platform made in Germany. Our cloud servers are compliant with the European General Data Protection Regulation (GDPR). They are located in Germany.

### Headless Commerce 

The headless commerce architecture provides the decoupling of front-end user interfaces from back-end solutions. This allows all types of front ends ("heads") to evolve independently. As a result, it gives e-commerce executives the flexibility and freedom to create e-commerce experiences which are perfectly aligned with their business models and partners. 

> - Arbeiten in Listen (

Author: ```suggestion

```

> - Arbeiten in Listen (

Author: ```suggestion

```

>     - Sortieren (auf-absteigend), 

Author: ```suggestion
 
```

>     - Filter

Author: ```suggestion

```

# Innovative MACH+ architecture

The MACH+ approach is considered as fundamental for the newest generation of software and systems integrators to quickly adapt new evolutions, trends, and technologies. The *Actindo Core1 Platform* offers maximum flexibility, highest performance, and future proofing by using the four guiding principles of microservices, API-first, cloud-native, and headless. 

### Microservices

> With the help of pre-built commerce building blocks, your teams can easily create a customer-specific infrastructure. Furthermore, the teams can agilely scale and evolve the IT-framework as needed using a low-code approach. If you have a PWA (Progressive Web Apps) online shop or custom web interfaces and apps for your sales teams, you can quickly implement and improve any commerce solution. 

Reviewer: ```suggestion
With the help of pre-built e-commerce building blocks, your teams can easily create a customer-specific infrastructure. Furthermore, the teams can agilely scale and evolve the IT-framework as needed using a low-code approach. If you have a PWA (Progressive Web Apps) online shop or custom web interfaces and apps for your sales teams, you can quickly implement and improve any commerce solution. 
```

### Flexible APIs

> Companies can accelerate innovation with modern APIs (Advanced Programming Interfaces) and easy-to-use low-code development environment. With 500+ API endpoints, you can integrate existing backend systems or additional third-party applications. You can extend the platform to your needs &ndash; regardless of the front end of your customer's devices. 

Reviewer: API = application programming interface?

> Companies can accelerate innovation with modern APIs (Advanced Programming Interfaces) and easy-to-use low-code development environment. With 500+ API endpoints, you can integrate existing backend systems or additional third-party applications. You can extend the platform to your needs &ndash; regardless of the front end of your customer's devices. 

Reviewer: ```suggestion
Companies can accelerate innovation with modern APIs (application programming interfaces) and easy-to-use low-code development environment. With over/more than 500 API endpoints, you can integrate existing backend systems or additional third-party applications. You can extend the platform to your needs &ndash; regardless of the front end of your customer's devices. 
```
API = application programming interface?

### Cloud technology

> Instead of worrying about maintaining, connecting, and updating a monolithic ERP software, you can use a fail-safe and scalable platform made in Germany. Our cloud servers are compliant with the European General Data Protection Regulation (GDPR). They are located in Germany.

Reviewer: Vorschlag:  Our cloud servers are compliant with the European General Data Protection Regulation (GDPR), as they are located in Germany. (oder As they are located in Germany, our cloud servers...)

> ### Headless Commerce 

Reviewer: ```suggestion
### Headless e-commerce 
```

> The headless commerce architecture provides the decoupling of front-end user interfaces from back-end solutions. This allows all types of front ends ("heads") to evolve independently. As a result, it gives commerce executives the flexibility and freedom to create commerce experiences which are perfectly aligned with their business models and partners. 

Reviewer: ```suggestion
The headless commerce architecture provides the decoupling of front-end user interfaces from back-end solutions. This allows all types of front ends ("heads") to evolve independently. As a result, it gives e-commerce executives the flexibility and freedom to create e-commerce experiences which are perfectly aligned with their business models and partners. 
```
Generell all "commerce" durch "e-commerce" ersetzen?

> <!---Weitere Themen

Author: ```suggestion

```
